demo/index.html

```diff
@@ -38,6 +38,7 @@
   selector.addEventListener('change', function() {
     document.documentElement.lang = this.value;
     I18nMsg.lang = this.value;
+    Platform.performMicrotaskCheckpoint();
```

Comment:
I tried to make the change you suggested but I couldn't a method called performMicrotaskCheckpoint inside of the WebComponents object. I also tried searching for performMicrotaskCheckpoint inside webcomponents.js and I couldn't find it.